Replace vManage with Manager (#451)
CiscoDevNet/catalystwan: README.md

[![Python-Supported](https://img.shields.io/static/v1?label=Python&logo=Python&color=3776AB&message=3.8%20|%203.9%20|%203.10%20|%203.11%20|%203.12)](https://www.python.org/)

> Cisco Catalyst SD-WAN SDK is a package for creating simple and parallel automatic requests via official Manager API. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being run in, you just need a connection to any Manager.

Reviewer: ```suggestion
Cisco Catalyst WAN SDK is a package for creating simple and parallel automatic requests via official Manager API. It is intended to serve as a multiple session handler (provider, provider as a tenant, tenant). The library is not dependent on environment which is being run in, you just need a connection to any Manager.
```

## Installation
```console
pip install catalystwan
```

## Session usage example

> Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with Manager. We provide ready to use authenticetion, you have to simply provide the Manager url, username and password as as if you were doing it through a GUI. 

Reviewer: ```suggestion
Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with SDWAN Manager. We provide ready to use authentication, you have to simply provide the Manager url, username and password as as if you were doing it through a GUI. 
```

```python
from catalystwan.session import create_manager_session

url = "example.com"
username = "admin"
password = "password123"

> session = create_manager_session(url=url, username=username, password=password)

Reviewer: Use context manager as suggested method of using session
```suggestion
with create_manager_session(url=url, username=username, password=password) as session:
    session.get("/dataservice/device")
```
Explain that `close()` method needs to be executed to release user session resource on SDWAN manager, when not using context manager.
 ```suggestion
session = create_manager_session(url=url, username=username, password=password)
session.get("/dataservice/device")
session.close()
```
